scripts/configs/template.conf

```diff
@@ -33,10 +35,13 @@ SampleNDRange = # currently unsupported
 
 [Harness]
 Name = harness_stencil
-Args = -p 2 -s 1024 
+Args = -s 1024
+#partially moved to environment.conf
```

Comment:
envirenment.conf sepcifies the openCL plattform and device to use for execution. These arguments are tied to the openCL setup so they were moved to the environment.conf.
Additional arguments that are tied to the exploration as for example "-s 1024" can still be set using the [Harness] Args field here.
That's what I meant with "partitially moved".